ci/custom_stemcell/tasks/test-stemcell.sh

```diff
@@ -1,22 +1,44 @@
 #!/usr/bin/env bash
 
-set -eu
+set -x
 
-export BOSH_BINARY_PATH=$(realpath bosh-cli/bosh-cli-*)
-chmod +x $BOSH_BINARY_PATH
-export BOSH_ENVIRONMENT=`$BOSH_BINARY_PATH int director-state/director-creds.yml --path /internal_ip`
-export BOSH_CA_CERT=`$BOSH_BINARY_PATH int director-state/director-creds.yml --path /director_ssl/ca`
-export BOSH_CLIENT=admin
-export BOSH_CLIENT_SECRET=`$BOSH_BINARY_PATH int director-state/director-creds.yml --path /admin_password`
-export SYSLOG_RELEASE_PATH=$(realpath syslog-release/*.tgz)
-export STEMCELL_PATH=$(realpath stemcell/*.tgz)
-export BOSH_stemcell_version=\"$(realpath stemcell/version | xargs -n 1 cat)\"
+echo "Login to bosh-lite"
+bosh2 envs
+echo "environments:">>~/.bosh/config
+echo "- url: ${DIRECTOR_HOST}">>~/.bosh/config
+echo "  ca_cert: |">>~/.bosh/config
+echo "    -----BEGIN CERTIFICATE-----">>~/.bosh/config
+echo "    ${CA_CERT}" >>~/.bosh/config
+echo "    -----END CERTIFICATE-----">>~/.bosh/config
+echo "  alias: ${ALIAS}" >>~/.bosh/config
+echo "  username: admin" >>~/.bosh/config
+echo "  password: ${PASSWORD}">>~/.bosh/config
+stemcell_version=$(cat version/number | sed 's/\.0$//;s/\.0$//')
+stemcell_filename=light-bosh-stemcell-${stemcell_version}-${IAAS}-${HYPERVISOR}-${OS_NAME}-${OS_VERSION}-go_agent.tgz
+bosh2 -e ${ALIAS} deld -d custom-stemcell -n
+bosh2 -e ${ALIAS} delete-stemcell bosh-bluemix-xen-ubuntu-xenial-go_agent/${stemcell_version} -n
+bosh2 -e ${ALIAS} upload-stemcell https://s3.us.cloud-object-storage.appdomain.cloud/${bucket}/${stemcell_filename}
+bosh2 -e ${ALIAS} upload-release http://${filew3_server}/releases/security-release/${RELEASE}/${RELEASE}-security-release.tgz
 
-pushd bosh-linux-stemcell-builder
-  export PATH=/usr/local/go/bin:$PATH
-  export GOPATH=$(pwd)
+if [ -e "deploy-yml/yml/custom-stemcell.yml" ]; then
+  bosh2 -e ${ALIAS} -d custom-stemcell deploy deploy-yml/yml/custom-stemcell.yml -n
+fi
```

Comment:
What happen if `custom-stemcell.yml` is not exist

Reply:
```
if [ -e "deploy-yml/yml/custom-stemcell.yml" ]; then
  bosh2 -e ${ALIAS} -d custom-stemcell deploy deploy-yml/yml/custom-stemcell.yml -n
else
 echo "Deploy vm failed since the deployment yml file does not exist "
 exit 1
fi
```